src/main.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+main() {
+    args=( $(find ./src/programs -type f | sed 's!.*/!!' | tr '\n' ' ') "${@}" )
```

Comment:
**[shellcheck]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
Prefer mapfile or read -a to split command output (or quote to avoid splitting). [SC2207](https://github.com/koalaman/shellcheck/wiki/SC2207)